### PR TITLE
Add collapse menu

### PR DESCRIPTION
**Pull request by Ewoskyx**
## Collapse Menu

In this project we did ;
- add main.js for the collapse menu
- update style.css for collapse menu
- update index.html for collapse menu

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const toggleIcon = document.getElementById('toggleIcon');
+const toggleElement = document.getElementById('dropdownClick');
```
Comment (dasileker) [inline main.js:2]: - [ ] When I switch to the mobile version the menu is dropdown.

jjjj

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Ewoskyx,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (Ewoskyx) [reply main.js:2]: @dasileker  Thank you for the review and suggestion. It has taken a lot of time to figure out but finally, I did it 😃

**Review by ignatius22 — APPROVED** [review]
## Hi, @Ewoskyx , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**